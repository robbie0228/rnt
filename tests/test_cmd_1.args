-link2 V3D1D2V4V2D3D4V1
-link1 V4V1V2V3D1D4D3D2
-ability1 USLDP
-ability2 AAWFF
