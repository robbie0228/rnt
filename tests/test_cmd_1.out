Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: V4   b: V1   c: V2   d: V3
e: D1   f: D4   g: D3   h: D2
========
abcSSfgh
...de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1's Abilities:
1: Uber, Unused
2: Scan, Unused
3: Link Boost, Unused
4: Download, Unused
5: Polarize, Unused

Player 2's Abilities:
1: Cop, Unused
2: Cop, Unused
3: Whey, Unused
4: Firewall, Unused
5: Firewall, Unused

Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: ?    b: ?    c: ?    d: ?
e: ?    f: ?    g: ?    h: ?
========
.bcSSfgh
a..de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: V3   B: D1   C: D2   D: V4
E: V2   F: D3   G: D4   H: V1

